source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -415,6 +417,32 @@ This option sets the frequency with which the Windows agent will check that the
 | **Allowed values** | A positive number, time in seconds |
 +--------------------+------------------------------------+
 
+whodata
+^^^^^^^
+
+Inside this tag we are going to define the Whodata options.
+
++--------------------+------------------------------------+
+| **Default value**  | N/A                                |
++--------------------+------------------------------------+
+
+audit_extra_key
```

Comment:
Please, move this tag to indicate that it goes inside the _whodata_ tag. In this way it looks like an independent label.